Neos.ContentRepository.Core/Classes/SharedModel/Node/NodeAggregateId.php

```diff
@@ -50,6 +50,23 @@ public static function fromString(string $value): self
         return new self($value);
     }
 
+    /**
+     * Calculates a deterministic nodeAggregateId.
+     *
+     * Used for determining the nodeAggregateIds of tethered nodes.
+     *
+     * e.g. in case you want to get the main content collection node aggregate id:
```

Comment:
```suggestion
     * e.g. in case you want to get the main content collection node aggregate id:
     *
     * NOTE: Previously ids of tethered nodes were _not_ deterministic, so this method should not be used to look up tethered nodes
     * Instead {@see \Neos\ContentRepository\Core\Projection\ContentGraph\ContentSubgraphInterface::findChildNodeConnectedThroughEdgeName()} can be used
```